doc/chapters/api/index.rst

```diff
@@ -24,88 +13,276 @@ Methods
 
 Ping
 ----
-Ping()
+Controls availability of com.pelagicore.SoftwareContainerAgent interface
+
+:Parameters:
+        - **None**
+
+:Return Value:
+        - **None** 
+
+:Example Usage:
+
+::
+
+        dbus-send --system --print-reply \
+        --dest=com.pelagicore.SoftwareContainerAgent \
+        /com/pelagicore/SoftwareContainerAgent \
+        com.pelagicore.SoftwareContainerAgent.Ping
 
 CreateContainer
 ---------------
-:containerIDPrefix: string
-:config: string
+Creates a new container and returns created container id.
+
+:Parameters:
+        - **IDPrefix** : ``string`` prefix for random container id.
+        - **config** : ``string`` config file in json format.
+        
+                Example config JSON::
+        
+                [{"enableWriteBuffer": true}]
+
+ 
+
+:Return Value:
+        - **containerID** ``uint32`` ID of created software container.
+
+
 
-uint32 containerID CreateContainer(string containerIDPrefix, string config)
+:Example Usage:
+
+::
+
+        dbus-send --system --print-reply \
+        --dest=com.pelagicore.SoftwareContainerAgent \
+        /com/pelagicore/SoftwareContainerAgent \
+        com.pelagicore.SoftwareContainerAgent.CreateContainer \
+        string:"00" string:'[{"enableWriteBuffer": true}]'
```

Comment:
My comment about four lines is obviously valid here as well (even though there are five lines)